scripts/airdrop.js

```diff
@@ -1,4 +1,6 @@
 const fs = require("fs").promises
+const csv = require("csv-parser")
+const BigNumber = require("bignumber.js")
```

Comment:
I think this package is also missing an explicit dependency in `package.json`. The reason why this may be important is that if you don't add an explicit dependency, and one of your other dependencies stop using this package internally, then this will stop working out of nowhere.

Reply:
I remember we had a bug in this repo for this very reason: one of our file was importing `bn.js` but we didn't explicitly mark it as a dependency in `package.json`. The result was that because of that we were using an old version of `bn.js` imported by another package which had a subtle but critical bug (bad input parsing on `BN` creation) that was quite hard to debug.